src/latexify/latexify_visitor_test.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 Google LLC
```

Comment:
This file looks to contain only end-to-end tests. It may be good to either:
- Move it into `src/integration_tests`
- Rewrite tests to directly call `LatexifyVisitor`.